MyPortfolio.services.api/Program.cs

```diff
@@ -0,0 +1,22 @@
+// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace MyPortfolio.Services.api
+{
+    public class Program
+    {
+        public static void Main(string[] args) =>
+         CreateHostBuilder(args).Build().Run();
```

Comment:
Fix the indentation.

Reply:
Since this is a CODE RUB task, I thought I can do it parallel while my Init API in review stage.
So, I have to start next task, only after my previous PR is closed?
Please confirm my understanding.

Reply:
Now take the latest from main and merge to this PR so show only the difference for this PR.